### DISCUSSION TITLE
Default servers seem to still be using a pretty old version

### DISCUSSION THREAD

**Discussion by powerjungle** [Syncplay/syncplay#693] [Ideas]
The servers shown when installing syncplay, for example "syncplay.pl:8995", seem to not have been updated for a while. The reason I noticed that is because of a commit and discussion from some time ago.

https://github.com/Syncplay/syncplay/commit/619dc3abd70bb14492093ca21c0126d23748295e

https://github.com/Syncplay/syncplay/discussions/596

That should mean the usernames longer than 16 characters would be shortened, but they're not, which leads me to believe they're still using a version that's older than 1.7.0.

Are they forgotten or are they using a distro like Debian Bookworm? I feel like it would be nice for the default servers to be with the same versions as the currently released clients.

**Comment by Et0h** [comment]
You're right, we were using an old version - thanks for letting us know. We have now updated to 1.7.3.
